tests/unit/utils.py

```diff
@@ -11,3 +12,18 @@ def get_data(name, data_dir="data"):
     with open(path, "r") as file_data:
         data = json.load(file_data)
     return data
+
+
+def copy_as_temp(data_path, text=True):
+    """
+    Makes a temporary copy of file relative to test dir and return it file path.
+    """
+    source_path = os.path.join(test_dir_path(), data_path)
+    temp_fp, temp_path = tempfile.mkstemp(text=text)
+
+    with open(temp_fp, "w") as out_f:
```

Comment:
Possible tip: I *think* you can use [`shutil.copy`](https://docs.python.org/3.8/library/shutil.html#shutil.copy), no need to open the file.

Reply:
Fixed. Funny thing, wanted to use it originally, but then I though that `mkstemp` will return an open file. It doesn't, I fixed it by adding open instead of returning back.